Update WPT tests for referenceTarget

alice opened the pull request:
#### 8081f52350093a8874ef1d0d58d89cc18f5c458a
<pre>
Update WPT tests for referenceTarget
<a href="https://bugs.webkit.org/show_bug.cgi?id=293900">https://bugs.webkit.org/show_bug.cgi?id=293900</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/3778a08fe93125455c450bbc480ced3452cac59e">https://github.com/web-platform-tests/wpt/commit/3778a08fe93125455c450bbc480ced3452cac59e</a>

* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/commandfor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/commandfor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/form-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/form.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/interesttarget.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/interesttarget.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/label-for-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/label-for.html:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-idl-setters-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-idl-setters.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-imperative-setup-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-imperative-setup.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection.html:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/resources/property-reflection-helper.js: Added.
(test_property_reflection):
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/297147@main">https://commits.webkit.org/297147@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/21b1b2f17c0910eefb85d820c536e1d60ae9f003

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/106014 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/25763 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/16157 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/111210 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/56610 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/26417 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/34266 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/80530 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/109018 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/20849 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/95676 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/60851 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/20451 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/13773 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/56048 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/90224 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/13809 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/114066 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/33152 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/24465 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/89607 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/33516 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/91904 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/89297 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23687 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/34161 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/11975 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/28711 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/33077 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/32823 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/36172 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/34421 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (changes requested):
Can you remove the second commit? Seems fine with that changed, modulo bots.

annevk reviewed (approved):
Looks good modulo bots.

alice committed:
Update WPT tests for referenceTarget
https://bugs.webkit.org/show_bug.cgi?id=293900

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3778a08fe93125455c450bbc480ced3452cac59e

* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/commandfor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/commandfor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/form-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/form.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/interesttarget.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/interesttarget.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/label-for-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/label-for.html:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-idl-setters-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-idl-setters.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-imperative-setup-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-imperative-setup.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection.html:
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/resources/property-reflection-helper.js: Added.
(test_property_reflection):
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/w3c-import.log:

Canonical link: https://commits.webkit.org/297147@main
